react/w-books/src/app/components/Link/index.tsx

```diff
@@ -0,0 +1,29 @@
+import clsx from 'clsx';
+import React, { ReactNode } from 'react';
+import { Link } from 'react-router-dom';
+
+interface LinkButtonProps {
+  children: ReactNode;
+  className?: string;
+  isFilled?: boolean;
+  isWidthAuto?: boolean;
+  to?: string;
+}
+
+export default function LinkButton({ children, className, isFilled, isWidthAuto, to }: LinkButtonProps) {
+
+  return (
+    <Link
+      to={to ?? '/'}
+      className={
+        clsx(
+          'button',
+          className,
+          {
+            ['filled']: isFilled,
+            ['auto-width']: isWidthAuto
+          }
+        )}
+    > {children}</Link >
+  );
+}
```

Comment:
I think this component is just a name wrapper. What logic are you abstracting here? If you are asbtracting just the scss class, i think with a global scss it would be enought